vNext/test/Tester/Tester.csproj

```diff
@@ -57,6 +57,39 @@
       <Project>{40ee3b00-d381-485f-9c69-ff706837deed}</Project>
       <Name>OrleansTestingHost</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Orleans\Orleans.csproj">
```

Comment:
No. Tester is empty in vNext. I plan to port over other tests which are not using HostedTestClusterEnsureDefaultStarted  into another project. 

Yeah haha. I guess it just difference of couple days. those tests are separately pretty fast. so there there.